package.json

```diff
@@ -15,6 +15,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.1",
```

Comment:
should be included in @babel/preset-env

Reply:
https://babeljs.io/docs/en/babel-preset-env#how-does-it-work

> It is important to note that @babel/preset-env does not support stage-x plugins.